engine/src/test/java/org/hibernate/search/test/backend/lucene/ScheduledCommitPolicyTest.java

```diff
@@ -35,6 +37,7 @@
  * @author gustavonalle
  */
 @RunWith(BMUnitRunner.class)
+@Category(SkipOnElasticsearch.class)
```

Comment:
We usually add a comment for this. Just a comment stating that this test is specific to Lucene would do.